public/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Steam</title>
+</head>
+<body style="background: #1b2837; font-family: Arial, Helvetica, sans-serif; color: #c6d4df;">
+  <div id="app"></div>
```

Comment:
I would suggest changing the name of your root div id in your service to "metadata" like you did below if you didn't already, so when you run all of our bundles on your proxy, it is clear whose is whose.